integration_test/cases/browser/set_value_test.exs

```diff
@@ -0,0 +1,71 @@
+defmodule Wallaby.Integration.Browser.SetValueTest do
+  use Wallaby.Integration.SessionCase, async: true
+
+  setup %{session: session} do
+    page = visit(session, "forms.html")
+    {:ok, %{page: page}}
+  end
+
+  describe "set_value/3" do
+    test "accepts a query", %{page: page} do
+      page
+      |> send_keys(Query.text_field("Name"), ["Chris", :tab, "c@keathley.io"])
+
+      assert page
+      |> find(Query.text_field("Name"))
+      |> has_value?("Chris")
+
+      assert page
+      |> find(Query.text_field("email"))
+      |> has_value?("c@keathley.io")
```

Comment:
I addressed this in response to your other point. Should I just remove this test or try to make it fit set_value?